lib/init.g

```diff
@@ -1018,8 +1030,11 @@ end );
 
 if IsHPCGAP and THREAD_UI() then
   ReadLib("hpc/consoleui.g");
-  MULTI_SESSION();
-else
+  if IsLIBGAP = false then
```

Comment:
Hmm... I was confused by the new logic for a moment (sorry for missing it in the previous version)... How about this
```gap
if IsLIBGAP then
   # GAP is used as a library, so don't start a session
elif IsHPCGAP and THREAD_UI() then
  MULTI_SESSION();
else
  SESSION();
fi;
```

Reply:
I am not sure actually. If we ever decide to have a HPC-GAP libGAP, the way it is now we could leave this as it is.

Reply:
On the otherhand, nevermind. I will change it to do have this logic.